src/sensitivities/chainrules.jl

```diff
@@ -147,6 +148,17 @@ function should_use_rrule(sig)
         Tuple{typeof(sum),typeof(abs2),AbstractArray},
     } && return false
 
+
+    # Functions that cause Nabla to have issues and that we don't use
+    sig <: Union{
+        Tuple{Type{<:Array}, AbstractArray},  # Nabla support for constructors is limitted
+    } && return false
+
+    opT ∈ typeof.((
+        Base.vect,  # currently having an issue with this being defined twice.
+                    # TODO: debug why and if ever we need this
```

Comment:
Maybe add a link to an issue or this PR?

Reply:
This special case isn't needed anymore so i have deleted it.